src/ray/core_worker/gcs_server_address_updater.cc

```diff
@@ -64,8 +64,23 @@ void GcsServerAddressUpdater::UpdateGcsServerAddress() {
              "the worker might crash. Connection status "
           << status;
       if (failed_ping_count_ == RayConfig::instance().ping_gcs_rpc_server_max_retries()) {
-        RAY_LOG(FATAL) << "Failed to receive the GCS address from the raylet for "
-                       << failed_ping_count_ << " times. Killing itself.";
+        std::stringstream os;
```

Comment:
consider using https://abseil.io/docs/cpp/guides/format

Reply:
Looks like we are using std::stringstream everywhere already. Do you think we should migrate to use this instead?

Reply:
If you prefer to use this, I can make a separate PR to make all `stringstream` to `absl:: StrFormat` instead of handing it locally here.